### PR TITLE
update FlxMouseEventManager class doc

### PR DESCRIPTION
**Pull request by T1mL3arn**

### CODE REVIEW

File: flixel/input/mouse/FlxMouseEventManager.hx [hunk 1]
```diff
@@ -38,6 +32,17 @@ import flixel.util.FlxDestroyUtil;
  * function onMouseOver(object:FlxObject) {}
  * function onMouseOut(object:FlxObject) {}
  * ```
+ *
```
Comment (T1mL3arn) [inline flixel/input/mouse/FlxMouseEventManager.hx:34]: Doc below this comment probably should be moved to `FlxMouseEvent`, but let it be here for now.

Comment (Geokureli) [reply flixel/input/mouse/FlxMouseEventManager.hx:34]: Why not do it now?

File: flixel/input/mouse/FlxMouseEventManager.hx [hunk 2]
```diff
@@ -6,28 +6,21 @@ import flixel.FlxBasic;
...
 import flixel.input.mouse.FlxMouseButton.FlxMouseButtonID;
 import flixel.math.FlxPoint;
 import flixel.util.FlxDestroyUtil;
 
 /**
  * Provides mouse event detection for `FlxObject` and `FlxSprite` (pixel-perfect for those).
- * To use it, initialize the manager and register objects / sprites.
+ * The manager get initialized by `Flixel`. To use it, get the manager with 
+ * [`FlxG.plugins.get()`](https://api.haxeflixel.com/flixel/system/frontEnds/PluginFrontEnd.html#get) 
+ * and register objects / sprites:
```
Comment (Geokureli) [inline flixel/input/mouse/FlxMouseEventManager.hx:18]: the static `FlxMouseEvent` class is meant to be the access point for the global manager, not via `FlxG.plugins`. It's Similar to `FlxTween` or `FlxTimer`. All of this doc should probably be moved there

**Review by Geokureli — CHANGES_REQUESTED** [review]
Move all of this to `FlxMouseEvent`.  FlxMouseEventManager should really just tell them that this global managing functionality was moved there, and that a new instance can be made for private management (but isn't recommended)

**Comment by Geokureli** [comment]
Thanks!

**Pull request merged**